### DISCUSSION TITLE
Why doesn't photo prism pick up faces from picasa and TagThatPhoto embedded in the EXIF

### DISCUSSION THREAD

**Discussion by cybernard** [photoprism/photoprism#3461] [Ideas]
I have most of the faces labelled either using Picasa or TagThatPhoto, but your product didn't detect a signal face based on the EXIF.
I have 50,000+ photos scanned.

**Comment by graciousgrey** [comment]
Importing face tags written by other tools is planned:

- https://github.com/photoprism/photoprism/issues/747
- https://github.com/photoprism/photoprism/issues/554
- https://github.com/photoprism/photoprism/issues/1571
- https://github.com/photoprism/photoprism/issues/1570
